resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/submit/submitsteps/PythonStep.scala

```diff
@@ -27,15 +27,19 @@ private[spark] class PythonStep(
     filesDownloadPath: String) extends KubernetesSubmissionStep {
 
   override def prepareSubmission(driverSpec: KubernetesDriverSpec): KubernetesDriverSpec = {
+    val resolvedOtherPyFilesString = if (otherPyFiles.isEmpty) {
```

Comment:
@ifilonenko is this sufficient to cover the arguments handling? SPARK_DRIVER_ARGS is loaded in `BaseSubmissionStep` already. Or is there more we need to do here?